.github/workflows/_build_wamrc.yml

```diff
@@ -0,0 +1,90 @@
+# Copyright (C) 2019 Intel Corporation.  All rights reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+name: build wamrc
```

Comment:
Must the file name begin with `_`?

Reply:
just a habit. not necessary. Any suggestion?